### PR TITLE
Adds basic documentation for internal packages

### PR DESCRIPTION
**Pull request by ScreamingTaco**
Oddly enough, even when I ran `make fmt` and tried using GoLand when editing the files, the imports still did not organize in alphabetical order. I went back and changed them manually to reflect how they were originally.

I'll get started on more docs soon. Thanks again for the opportunity to help with such a cool project (albeit in a small way).

### CODE REVIEW

**Comment by ScreamingTaco** [comment]
I think I figured out why mine was changing the format in the first place. Besides running `go fmt`, my editor was also running `goimports` on the save of every file.

File: internal/commands/migrate.go [hunk 1]
```diff
@@ -12,6 +13,7 @@ var MigrateCommand = cli.Command{
 	Action: migrateAction,
 }
 
+// Automatically migrates/ initializes database; called by MigrateCommand
```
[the remark this answers is not in the document]
Comment (ScreamingTaco) [reply internal/commands/migrate.go]: Thanks! I didn’t catch that. Fixed in latest commit :)

**Comment by lastzero** [comment]
See issue #14 Source-code documentation (go doc)

File: internal/api/doc.go [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/*
+Package api contains REST request handlers used by the server package.
+
+See https://photoprism.org/ for more information about PhotoPrism.
+*/
```
Comment (pggee7198) [inline internal/api/doc.go:5]: Gzla

**Review by pggee7198 — COMMENTED** [review]
GzlaEorld

**Pull request merged**